### PR TITLE
Support kwargs in ai_edge_torch.convert to named inputs

### PR DESCRIPTION
**Pull request by chunnienc**
torch_xla does not support exported_program with kwargs to stablehlo, so I need to flatten inputs to a tuple before the very first torch.export.export.

BUG=https://b.corp.google.com/issues/350510246

### CODE REVIEW

File: ai_edge_torch/convert/conversion_utils.py [hunk 1]
```diff
@@ -47,8 +49,59 @@ class Signature:
...
   sample_args: tuple[torch.Tensor]
+  sample_kwargs: dict[str, torch.Tensor]
   dynamic_shapes: Optional[Union[Dict[str, Any], Tuple[Any]]] = None
 
+  @property
+  def _normalized_sample_args_kwargs(self):
+    args, kwargs = self.sample_args, self.sample_kwargs
+    if args is not None:
+      if not isinstance(args, tuple):
+        # TODO: Check value types
```
Comment (majiddadashi) [inline ai_edge_torch/convert/conversion_utils.py:60]: Please address these TODOs.

File: ai_edge_torch/convert/conversion_utils.py [hunk 2]
```diff
@@ -47,8 +49,59 @@ class Signature:
...
+    )
+
+    for name, value_spec in zip(dict_context, kwargs_spec.children_specs):
+      if value_spec.num_leaves == 1:
+        names.append(name)
+      else:
+        # value_spec.num_leaves may be greater than 1 when the value is a (nested)
+        # tuple of tensors. We haven't decided how we should support flattenable
+        # tensor containers  as inputs.
+        # TODO: Decide the behavior of tensor container as input (flatten or reject)
```
Comment (majiddadashi) [inline ai_edge_torch/convert/conversion_utils.py:96]: Please add a bug and the identifier next to the TODO.

File: ai_edge_torch/convert/conversion.py [hunk 3]
```diff
@@ -88,16 +88,14 @@ def convert_signatures(
   _warn_training_modules(signatures)
 
   exported_programs: torch.export.ExportedProgram = [
-      torch.export.export(
-          sig.module, sig.sample_args, dynamic_shapes=sig.dynamic_shapes
-      )
+      torch.export.export(sig.module, sig.flat_args, dynamic_shapes=sig.dynamic_shapes)
```
Comment (majiddadashi) [inline ai_edge_torch/convert/conversion.py:91]: why aren't we passing kwargs to `torch.export.export` as kwargs?

Comment (chunnienc) [reply ai_edge_torch/convert/conversion.py:91]: torch_xla does not support exported_program with kwargs to stablehlo, so I need to flatten all the inputs to a tuple before the very first torch.export.export.

**Review by majiddadashi — APPROVED** [review]

**Pull request merged**